dash/frontend/src/app/modules/private/pages/cluster/kube-bench/kube-bench.component.html

```diff
@@ -6,7 +6,7 @@
     <div class="page-card">
       <div class="row">
 
-        <div class="col-xs-12 col-sm-12 col-md-6 col-lg-3 margin-top-10" style="margin: 0; padding-left:0; height:150px; display: grid;!important; overflow: auto">
```

Comment:
When resized to small screens, are these tables scrollable?
I'd guess we'll have to add those classes (mentioned in a different comment)

Reply:
@becky-intelletive In the mobile size, info from these cards are visible (scrolls are not necessary)
<img width="713" alt="Screenshot 2023-07-24 at 12 04 18 PM" src="https://github.com/m9sweeper/m9sweeper/assets/56838011/8ba47fe2-4816-473a-8c36-0164ac81b2fc">
As for adding classes, are you referring to adding class="col-xs-12 col-sm-12 col-md-6 col-lg-3 margin-top-10" as a global class?

Reply:
(Same goes for the Kube Hunter page)

Reply:
If you're referring to the width of the 3 cards, Test status and vulnerabilities cards have been more narrow that way to accommodate Our Advice card. I can adjust Our Advice card to fix the cutoff text.

Reply:
That should be handled by the column sizing, which should be uniform for a small screen (col-xs-12 for all elements).